src/webmock/stub.cr

```diff
@@ -1,13 +1,13 @@
 class WebMock::Stub
   @method : String
-  @uri : URI
+  @uri : String | Regex
```

Comment:
Why change from `URI` to `String`? This adds a lot of overhead for parsing the string for every single `matches_uri?` call.

Reply:
Addressed in new commit. Thanks!